client/src/sections/resources/IndustrialFireExtinguisher.tsx

```diff
@@ -59,27 +59,29 @@ const IndustrialFireExtinguisher = () => {
       hide={!useHave($item`industrial fire extinguisher`) || foam <= 0}
     >
       <Line>
-        Remaining foam: {foam}. That's a total of {Math.min(foam / 10)} polar
-        vortices.
+        {`${foam} foam (${Math.floor(foam / 10)} polar
+        vortices).`}
       </Line>
-      {showBat && (
-        <Line>
-          You can use Constricted Blast to blown down a wall in the Bat Hole
-        </Line>
-      )}
-      {showKnob && (
-        <Line>You can use Foam the Place to obtain the Knob Harem Outfit</Line>
-      )}
-      {showCyrpt && (
-        <Line>
-          You can use Replace the Chill to undefile a zone by 10 evil.
-        </Line>
-      )}
-      {showBlech && (
-        <Line>You can use Cool it Down to advance your Blech House timer.</Line>
-      )}
-      {showDesert && (
-        <Line>You can use Take a Drink to get 15 turns of Ultrahydrated.</Line>
+      {foam >= 20 && (
+        <>
+          {showBat && (
+            <Line>
+              Use Constricted Blast to blown down a wall in the Bat Hole.
```

Comment:
for all of these, i would recommend using `<ListItem>` instead of `<Line>` since we're cleaning it up anyway. i would also recommend cutting down each of them to be formatted a bit differently. i think my ideal format here would be:

- **Constricted Blast:** +1 Bat Hole chamber unlocked
- **Foam the Place:** Obtain the Knob Harem Outfit
- **Replace the Chill:** Undefile a Cyrpt zone by 10 evil
- **Cool it Down:** Get 66% progress towards Blech House
- **Take a Drink:** 15 turns of Ultrahydrated

i'd also potentially consider adding an AdviceTooltipIcon to the bat hole one to explain the fun TES strat of "make sure you use a delayed combat effect to make it free.